css/main.scss

```diff
@@ -235,6 +235,10 @@ a.anchor {
     text-decoration: underline;
   }
 
+  .day {
+    margin-top: 20px;
```

Comment:
Shall we use the same 80px that the content is offset from the surrounding container?